packages/core-typings/src/IOAuthApps.ts

```diff
@@ -1,8 +1,11 @@
-export interface IOAuthApps {
-	_id: string;
+export interface IOAuthAppsInfo {
+	clientId: string;
 	name: string;
+}
```

Comment:
Personal pref: do we need this to be an interface? Can't we just `export type OauthAppsInfo = Pick<IOAuthApps, 'clientId' | 'name'>` ?

Reply:
I agree, that will be better.